git_machete/client.py

```diff
@@ -971,8 +972,23 @@ def print_line_prefix(branch_: LocalBranchShortName, suffix: str) -> None:
                         out.write(" %s%s%s\n" % (
                                   f"{dim(commit.short_hash)}  " if opt_list_commits_with_hashes else "", dim(commit.subject),
                                   fp_suffix))
-                elbow_ascii_only: Dict[str, str] = {AnsiEscapeCodes.DIM: "m-", AnsiEscapeCodes.RED: "x-", AnsiEscapeCodes.GREEN: "o-", AnsiEscapeCodes.YELLOW: "?-"}
-                elbow: str = u"└─" if not utils.ascii_only else elbow_ascii_only[edge_color[branch]]
+
+                elbow: str
+                if utils.ascii_only:
+                    elbow_ascii_only: Dict[str, str] = {
+                        AnsiEscapeCodes.DIM: "m-",
+                        AnsiEscapeCodes.RED: "x-",
+                        AnsiEscapeCodes.GREEN: "o-",
+                        AnsiEscapeCodes.YELLOW: "?-"}
+                    elbow = elbow_ascii_only[edge_color[branch]]
+                else:
+                    # Note that using the "├─"-style elbow has been considered as well:
```

Comment:
@kgadek now I reminded myself why I didn't choose to use three-legged elbows (as `tree` does) 4 years ago when I first implemented that:

![image](https://user-images.githubusercontent.com/3383210/166109344-ee6be4fe-abd1-4dd3-a5dc-be2c43213cf6.png)

This boundary b/w green and red doesn't look good :/